stdlib/Random/src/Random.jl

```diff
@@ -136,8 +136,13 @@ the amount of precomputation, if applicable.
 *types* and *values*, respectively. [`Random.SamplerSimple`](@ref) can be used to store
 pre-computed values without defining extra types for only this purpose.
 """
-Sampler(rng::AbstractRNG, x, r::Repetition=Val(Inf)) = Sampler(typeof(rng), x, r)
-Sampler(rng::AbstractRNG, ::Type{X}, r::Repetition=Val(Inf)) where {X} = Sampler(typeof(rng), X, r)
+Sampler(rng::AbstractRNG, x, r::Repetition=Val(Inf)) = Sampler(typeof_rng(rng), x, r)
+Sampler(rng::AbstractRNG, ::Type{X}, r::Repetition=Val(Inf)) where {X} =
+    Sampler(typeof_rng(rng), X, r)
+
+# GLOBAL_RNG currently represents a MersenneTwister
+typeof_rng(rng::AbstractRNG) = typeof(rng)
+typeof_rng(::_GLOBAL_RNG) = MersenneTwister
```

Comment:
`_GLOBAL_RNG` is not defined here yet.

Reply:
Ah right, is this why CI didn't build? I couldn't understand CI failures (am still used to travis interface).